src/microbe_stage/Microbe.cs

```diff
@@ -2545,6 +2545,12 @@ private void OnBodyExitedEngulfArea(Node body)
         }
     }
 
+    private bool CanBindToMicrobe(Microbe other)
+    {
+        // Cannot hijack the player, other species or other colonies (TODO: yet)
+        return other?.IsPlayerMicrobe == false && other.Colony == null && other.Species == Species;
```

Comment:
I think it would be clearer to have `if (other == null)` as a separate check first, so there wouldn't be a need to have this safe navigation operator used here `other?`

Reply:
Maybe even the entire null check should be removed as we are pretty sure right that touchedMicrobes does not contain any null entries (as I've been told there should already be check in the physics callback adding things to it to ensure it doesn't put a null in the list).

Reply:
`other` is actually extracted from a filtered list. If no touched microbe matches the filter, then `other == null`.

As for the use of `other?`, it has the merit to cover the cases of "improper" calls to this method (with null other). But, on the other hand, in such case we may want to just print an error in the code... or just let the game crash to remark it even more....

So, I'm happy with either solution.